### ISSUE TITLE
[pay with] Remove "retire using different payment" message

### ISSUE THREAD

**Issue by jabby09** [KlimaDAO/klimadao#1314]
On the RETIRE and BUY screens for pool listings, looking to update and standardize the user help message:

- [x] Remove the "To retire this project using a different form of payment, click here"
- [x] Note for the designs that the following copy was added in Production [Learn how to acquire USDC on Polygon.](https://docs.carbonmark.com/get-started/how-to-get-usdc-or-matic)

So currently, this is what the user help message looks like for RETIRE:
<img width="389" alt="Screenshot 2023-07-13 at 9 39 21 AM" src="https://github.com/KlimaDAO/klimadao/assets/103535195/6c3e8125-071a-4f26-9721-063d1d9bfa98">

So currently, this is what the user help message looks like for BUY:
<img width="408" alt="Screenshot 2023-07-13 at 9 41 25 AM" src="https://github.com/KlimaDAO/klimadao/assets/103535195/1a95bd16-a846-4ff1-b72c-94a567aef9bc">

In summary, the updated user messages should be:

**RETIRE**
Currently, Carbonmark only accepts Polygon USDC or Credit Card payments. [Learn how to acquire USDC on Polygon.](https://docs.carbonmark.com/get-started/how-to-get-usdc-or-matic)

**BUY**
Currently, Carbonmark only accepts Polygon USDC payments. [Learn how to acquire USDC on Polygon.](https://docs.carbonmark.com/get-started/how-to-get-usdc-or-matic)

**Comment by theprofessora** [comment]
done. source component updated here: https://www.figma.com/file/0vb829fJKYWWKszeSOvTTN?node-id=0:1&mode=design#475442520

**Comment by jabby09** [comment]
> done. source component updated here

@0xMakka see updated component, the forms need to be updated with these user messages:

RETIRE
Currently, Carbonmark only accepts Polygon USDC or Credit Card payments. [Learn how to acquire USDC on Polygon.](https://docs.carbonmark.com/get-started/how-to-get-usdc-or-matic)

BUY
Currently, Carbonmark only accepts Polygon USDC payments. [Learn how to acquire USDC on Polygon.](https://docs.carbonmark.com/get-started/how-to-get-usdc-or-matic)
